### PR TITLE
Add ftests

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

File: .github/workflows/unittests.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+jobs:
+  test:
+    name: Run JDK ${{ matrix.jdk-version }} tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk-version: [ "19" ]
```
Comment (ligangty) [inline .github/workflows/unittests.yaml:19]: better to use 11 because our prod env is using 11 now.

**Review by ligangty — COMMENTED** [review]
Seems CI can not pass due to cassandra issue

File: .github/workflows/unittests.yaml (hunk 1 above)
Comment (hjmodi) [reply .github/workflows/unittests.yaml]: added jdk 11

**Comment by hjmodi** [comment]
> Seems CI can not pass due to cassandra issue

That is due to docker not being installed. I have put a condition on the test to only run when docker is available.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**